content/deprecated-components/markdown-viewer/react/deprecated.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: Markdown viewer
+description: Displays rendered Markdown and facilitates interaction.
+reactId: markdown_viewer
+reactStatus: deprecated
+---
+
+import ReactComponentPage from '~/src/components/react-component-page'
+import { graphql } from "gatsby"
+
+export const query = graphql`
+  query MarkdownViewerDeprecatedQuery($componentId: String! = "markdown_viewer", $parentPath: String! = "/deprecated-components/markdown-viewer", $status: String! = "deprecated") {
+    ...ReactComponentInfo
+  }
+`
+
+export default function Layout({data, children}) {
+  return <ReactComponentPage data={data} showImport={false} tocItems={[{url: "#import", title: "Import"}, {url: "#examples", title: "Examples"}]}>
+    {children}
+  </ReactComponentPage>
+}
+
+The `MarkdownViewer` displays rendered Markdown and handles interaction (link clicking and checkbox checking/unchecking) with that content.
```

Comment:
I think we can just link to Primer Recipes https://ui.githubapp.com/storybook/?path=/story/recipes-markdownviewer--default

Reply:
Is it possible someone is using this component in dotcom and needs to reference the deprecated docs?

Reply:
Not seeing it in [Primer Query](https://musical-adventure-wlr3n3k.pages.github.io/?name=markdownviewer)